app/jobs/course/user_deletion_job.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+class Course::UserDeletionJob < ApplicationJob
+  include TrackableJob
+  queue_as :highest
```

Comment:
This is not a high priority task. Push to `default` queue.  
Only user-triggered autograding/file-download/password/email-confirmation related should be highest.

Reply:
done